test-network-function/lifecycle/suite.go

```diff
@@ -314,36 +314,36 @@ func testPodsRecreation(env *config.TestEnvironment) {
 			if len(deployments) == 0 {
 				return
 			}
+			if _, exists := namespaces[podNamespace]; !exists {
+				namespaces[podNamespace] = true
+			}
```

Comment:
You'll repeat the getDeploymentsNode(podNamespace) on line 326 for each pod in the same namespace. Is it necessary? I suggest adding here this:
```
else {
    continue
}
```

Reply:
right. will fix it